src/pages/api/_libs/airtable.ts

```diff
@@ -0,0 +1,54 @@
+import md5 from 'md5'
+import axios, { Axios } from 'axios'
+import redis from './redis'
+
+export interface AirtableOptions {
+  baseId: string
+  apiKey: string
+  expire?: number // seconds
+}
+
+export class Airtable {
+  private client: Axios
+
+  private expire: number
+
+  constructor({ baseId, apiKey, expire = 60 * 60 * 24 }: AirtableOptions) {
```

Comment:
24 hours expires? Feel too long

Reply:
We can refresh cache through Airtable, so a long expire time should be ok.
<img width="866" alt="Screen Shot 2022-08-04 at 2 25 03 PM" src="https://user-images.githubusercontent.com/106942446/182778156-c8858c94-64db-40fe-880b-728cae85daf2.png">